lib/helpers/is-valid-config-object.js

```diff
@@ -0,0 +1,34 @@
+export default function isValidConfigObjectFormat(config, POSSIBLE_CONFIG) {
+  const validLiteralValues = [true, false, 'warn', 'error'];
+  if (validLiteralValues.includes(config)) {
+    return true;
+  }
+
+  if (config === null && typeof config !== 'undefined') {
```

Comment:
Looks like the second condition might be unnecessary (since if config is `null` it’s guaranteed to not be `undefined` also)
```suggestion
  if (config === null) {
```